src/components/submission/table.vue

```diff
@@ -90,34 +120,36 @@ export default {
       return this.project != null && this.project.permits('submission.update');
     }
   },
-  mounted() {
```

Comment:
Previously, I manually added event listeners in the `mounted` hook instead of using Vue event handling. I thought I had to do that in order to use event delegation, but it turns out that that's not the case, so I've switched things over to Vue event handling. We have more event listeners now, so it's nice to be able to use the Vue event handling. Other than that change, most of the actual logic should be the same.